<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandWindow</class>
 <widget class="QMainWindow" name="CommandWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLineEdit" name="SetCorrALineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="SetVdbsLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="SetVdbsPushButton">
        <property name="text">
         <string>Set Vdbs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="SetAPDPushButton">
        <property name="text">
         <string>Set APD</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="SetCorrAPushButton">
        <property name="text">
         <string>Set CorrA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="SetAPDLineEdit"/>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="SetCorrBPushButton">
        <property name="text">
         <string>Set CorrB</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="SetCorrBLineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>425</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
